examples.http

```diff
@@ -1,14 +0,0 @@
-:token := (shell-command-to-string "panoptes-token")
```

Comment:
i'd like to keep the docs more discoverable and in one place instead of keeping 2 files (Readme and this) up to date. 

I think the docs in readme cover this but happy to add another example section.

Reply:
Except that is example is actually directly runnable (with restclient.el), whereas the README isn't. But I see your point. 2/10 on this.

Reply:
If one has emacs and that dep installed sure :) 
I'd be happy to support these as they should be pretty static, I'll revert this delete. I do think it should have all end point examples though.